### PR TITLE
Use style from lscolors to render the rest of the filename

### PR DESCRIPTION
**Pull request by nibon7**
# Description

Related: https://github.com/nushell/nushell/pull/6556#issuecomment-1246681644

When `use_ls_colors` is enabled, use style from lscolors to render the rest of the filename.

# Tests

Make sure you've done the following:

- [ ] Add tests that cover your changes, either in the command examples, the crate/tests folder, or in the /tests folder.
- [ ] Try to think about corner cases and various ways how your changes could break. Cover them with tests.
- [ ] If adding tests is not possible, please document in the PR body a minimal example with steps on how to reproduce so one can verify your change works.

Make sure you've run and fixed any issues with these commands:

- [x] `cargo fmt --all -- --check` to check standard code formatting (`cargo fmt --all` applies these changes)
- [x] `cargo clippy --workspace --features=extra -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect` to check that you're using the standard code style
- [x] `cargo test --workspace --features=extra` to check that all the tests pass

### CODE REVIEW

File: crates/nu-command/src/filters/find.rs [hunk 1]
```diff
@@ -535,6 +540,47 @@ fn find_with_rest_and_highlight(
     }
 }
 
+fn to_nu_ansi_term_style(style: &LsColors_Style) -> Style {
```
Comment (fdncred) [inline crates/nu-command/src/filters/find.rs:543]: Not a blocker but where did this function come from? It looks familiar.

Comment (nibon7) [reply crates/nu-command/src/filters/find.rs:543]: reference `lscolors::Color::to_ansi_term_color`

https://github.com/sharkdp/lscolors/blob/5516bc727f66e7931a62585034ca8cc3ecf3f699/src/style.rs#L40-L64

Comment (fdncred) [reply crates/nu-command/src/filters/find.rs:543]: The comment is what triggered me to think I had seen this before. "Below items are a rough..."

**Comment by fdncred** [comment]
thanks.

**Comment by fdncred** [comment]
very nice fix 
Screen Shot 2022-09-15 at 6 15 06 AM

**Pull request merged**